contrib/k8s/examples/mysql/advanced/mysql-dbms-instance.yaml

```diff
@@ -1,15 +1,15 @@
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ServiceInstance
 metadata:
-  name: dbms-only-sqldb-instance
+  name: my-mysql-dbms-instance
```

Comment:
Did we want to standardize on either my-* or example-*?

Reply:
crap i thought I changed that. hold on.

Reply:
it was changed in code but for some reason I hadn't saved it!